src/renderer/features/flexible-multisig-create/model/__tests__/confirm-model.test.ts

```diff
@@ -0,0 +1,38 @@
+import { allSettled, fork } from 'effector';
+
+import { type Account, type Chain } from '@/shared/core';
+import { networkModel } from '@/entities/network';
+import { walletModel } from '@/entities/wallet';
+import { confirmModel } from '../confirm-model';
+
+import { initiatorWallet, signerWallet, testApi } from './mock';
+
+describe('features/flexible-multisig/model/confirm-model', () => {
```

Comment:
Let's use some meaningful name for tests, not file paths. Like `flexible multisig confirm`

Reply:
The test itself has a meaningful description but for the test suit it's pretty convenient to use a path since some components have similar names